README.md

```diff
@@ -0,0 +1,88 @@
+# SoundBubble
+
+###### Group Members: [Martin Palanca](https://github.com/martipal), [Sherry Zhang](https://github.com/veronicz), [Yancey Yang](https://github.com/Talos6)
+
+SoundBubble is a platform that leverages Spotify's public API to build logs of a user's recently played tracks and makes this data visible to other users. Users can create and join groups to refine which users' song logs appear on their feed, as well as rate songs using an 'thumbs up' and 'thumbs down' scale.
```

Comment:
Should we modify this to include commenting?

Reply:
I think so

Reply:
but the 436I web page already shows like this

Reply:
Do we want to post in Slack to let Steve know that we want to update our description? Or just update it here and include that in our demo?

Reply:
cuz this one is stretch, just update here and demo it should be fine